htdocs/stripe/class/stripe.class.php

```diff
@@ -392,7 +392,7 @@ public function getPaymentIntent($amount, $currency_code, $tag, $description = '
     		    "payment_method_types" => array("card"),
     		    "description" => $description,
     		    "statement_descriptor" => dol_trunc($tag, 10, 'right', 'UTF-8', 1),     // 22 chars that appears on bank receipt (company + description)
-    			//"save_payment_method" => true,
```

Comment:
Why removing save_payment_method = true ?
When it was not set, the credit card was not saved on stripe side.